lxd/api_internal.go

```diff
@@ -122,14 +122,6 @@ func internalImport(d *Daemon, r *http.Request) Response {
 		return BadRequest(fmt.Errorf("target is required"))
 	}
 
-	path := containerPath(name, false)
-	err := d.Storage.ContainerStart(name, path)
-	if err != nil {
-		return SmartError(err)
-	}
-
-	defer d.Storage.ContainerStop(name, path)
```

Comment:
Hm, so this is going to be a bit funny to handle. Basically, when the container got deleted from the containers and storage volumes database && its storage volume got unmounted we have no easy way of remounting it. But we need to remount it in order to retrieve `backup.yaml`. So some reverse-engineering is required. What I'm thinking about is: `os.Readlink(containersymlink)` --> parse out pool --> lookup driver for pool in pool db --> mount storage volume for container based on this information. This should only be a problem with `lvm` and `zfs` afaik.